src/h265_nal_unit_parser.cc

```diff
@@ -74,6 +74,20 @@ H265NalUnitHeaderParser::ParseNalUnitHeader(const uint8_t* data,
   return ParseNalUnitHeader(&bit_buffer);
 }
 
+bool H265NalUnitHeaderParser::ParseNalUnitType(const uint8_t* data, const size_t length,
+      NalUnitType& naluType) noexcept {
+  if (length == 0) {
+    return false;
+  }
+  rtc::BitBuffer bitBuffer(data, length);
+  auto naluHeader = ParseNalUnitHeader(&bitBuffer);
```

Comment:
the whole function is just running this, and checking whether the result is a valid object or nullptr. I don't think you need a new function for that

Reply:
Thanks for review! 

Added this function as a convenience for two reasons:
1. Not needing to create a bit buffer, because that's an implementation detail of the parser. For example, I can parse the nalu type without using a bit buffer ((data[0] & 0x3F) >> 1)
2. Return nalu type as enum instead of uint32_t currently set in the H265NalUnitHeaderParser::NalUnitHeaderState

Let me know if you still think it's not needed.